NGitLab/Impl/PipelineClient.cs

```diff
@@ -145,7 +145,7 @@ private string CreateSearchUrl(PipelineQuery query)
             if (query.Scope.HasValue)
                 queryEntries.Add("scope", query.Scope.Value.ToString());
             if (query.Status.HasValue)
-                queryEntries.Add("status", query.Status.Value.ToString());
+                queryEntries.Add("status", query.Status.Value.ToString().ToLower());
```

Comment:
As good practice, in most cases it's better to use `ToLowerInvariant()` than `ToLower()` (which is culture-sensitive).

Reply:
@louis-z We should add the BannedSymbol analyzer to ensure nobody uses this method (and a few other problematic methods)

Reply:
Good idea, I'll add it.